.tp-config.json

```diff
@@ -4,6 +4,10 @@
     "**/*.jar"
   ],
   "parameters": {
+    "generateOppositeSide": {
```

Comment:
choose better name here like `generateClient` or `isClient`

Reply:
I explain my position, why i couldn't name it "client". If we name it "client" some people will immediately think about "server", but it's not. We couldn't speak about "client" and "server", because AsyncAPI describe both sides, what application receives (server) and what it sends (client).

Reply:
fair point, but still `generateOppositeSide` is not the best name, so suggest something better or accept proposal and properly explain in description that is meant by `Client` 😄

Reply:
name changed to `inverseOperations`, i think it's most fair and clear description about what really does this flag.
Description contains more info and use case